content/docs/user-guide/concepts/dvc-files.md

```diff
@@ -0,0 +1,26 @@
+# DVC Files
```

Comment:
Not sure we need this in basic concepts. There's a guide for these. Also, the term is "DVC Metafiles" (referring to .dvc and dvc.yaml/lock files). But even extracting those, there's still DVC internal files & dirs that should stay in the existing guide.

So the question is should we/ is it worth it to move the metafiles out of the existing guide into here?

Reply:
> So the question is should we/ is it worth it to move the metafiles out of the existing guide into here?

Let's try to answer this after seeing how the other concepts shape up. Working on it.